tailwind.config.ts

```diff
@@ -1,7 +1,10 @@
 import type { Config } from "tailwindcss";
 
+import daisyui from "daisyui";
```

Comment:
Do we have a import order lint?Is it relevant? Because usually `import type` stays at the bottom of the imports.

Reply:
Currently, we use [biome](https://biomejs.dev/guides/getting-started/) as our linter and code formatter, and when we run it to format the code, the imports are not affected. 

Therefore, I believe we don't have a specific rule for sorting imports, even though import sorting is enabled in the config file.

Reply:
Biome can fix the imports too, just need to config it I guess

Reply:
I agree that we should enable import sorting on biome. Consistency in import ordering is important.

Also @Diegiwg, as a minor suggestion, I'd recommend renaming the `daisyui` variable to `daysiuiPlugin`.

Reply:
We already have Biome configured to "organize imports" as specified in the `biome.json` file:

![image](https://github.com/user-attachments/assets/c0a51211-ba10-419c-a80a-c6c11de7ae3a)

After reviewing Biome's documentation on import sorting rules, I couldn't find any specific rule regarding `type import`. Reference: https://biomejs.dev/analyzer/import-sorting/#how-imports-are-sorted

For this reason, I believe that the `import type { Config } from "tailwindcss";` is not being affected by Biome.

Reply:
> Also @Diegiwg, as a minor suggestion, I'd recommend renaming the daisyui variable to daysiuiPlugin

Made in: https://github.com/podcodar/webapp/pull/175/commits/2fe4738294bb4d0bf2ce3ccdd0c9a0b829d47486

Reply:
> For reference: https://biomejs.dev/linter/rules/use-import-type/

This discusses enforcing the use of `type import`, not their ordering in the import list—at least that's how I understood it.